plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/mypy/MypyAnalysis.java

```diff
@@ -342,7 +331,9 @@ public void afterRunProcess(String output, String errors, IExternalCodeAnalysisS
                 Log.log(e);
             }
         }
-        for (MessageInfo messageInfo : lineColToMessage.values()) {
+        for (
```

Comment:
Please revert the strange indent...

Reply:
Yup, I did it in https://github.com/fabioz/Pydev/pull/294/commits/2d92da641b74a6cb41665be162aed7caf1f490ce